pkg/fetch/vendir_test.go

```diff
@@ -1,15 +1,14 @@
 // Copyright 2022 VMware, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-package fetch_test
+package fetch
```

Comment:
generally i recommend to keep them isolated so that we have enough "public" interface to check on whatever we need to check.